packages/babel-parser/src/tokenizer/types.js

```diff
@@ -86,6 +86,7 @@ export const types: { [name: string]: TokenType } = {
   regexp: new TokenType("regexp", { startsExpr }),
   string: new TokenType("string", { startsExpr }),
   name: new TokenType("name", { startsExpr }),
+  primaryTopicReference: new TokenType("#", { startsExpr }),
```

Comment:
Why is this token needed? Can’t we just reuse tt.hash?

Reply:
On first blush, it conflicts with the private properties plugin, but I'll try to reuse that.